common_genes/1_common_DEGs_and_gene_groups.ipynb

```diff
@@ -0,0 +1,892 @@
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>PA2618_at</th>\n",
+       "      <td>97.566691</td>\n",
+       "      <td>90.007215</td>\n",
+       "      <td>PA2618</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA3479_rhlA_at</th>\n",
+       "      <td>85.940880</td>\n",
+       "      <td>98.890693</td>\n",
+       "      <td>PA3479</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA0512_at</th>\n",
+       "      <td>96.268926</td>\n",
+       "      <td>74.918831</td>\n",
+       "      <td>PA0512</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA3187_at</th>\n",
+       "      <td>88.626532</td>\n",
+       "      <td>87.626263</td>\n",
+       "      <td>PA3187</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA5191_at</th>\n",
+       "      <td>86.571738</td>\n",
+       "      <td>74.918831</td>\n",
+       "      <td>PA5191</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                Percentile (simulated)  prop DEGs gene id\n",
+       "ProbeID                                                  \n",
+       "PA2618_at                    97.566691  90.007215  PA2618\n",
+       "PA3479_rhlA_at               85.940880  98.890693  PA3479\n",
+       "PA0512_at                    96.268926  74.918831  PA0512\n",
+       "PA3187_at                    88.626532  87.626263  PA3187\n",
+       "PA5191_at                    86.571738  74.918831  PA5191"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "print(common_DEGs.shape)\n",
+    "common_DEGs.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7454af42",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "common_DEGs = common_DEGs.set_index(\"gene id\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d35aa66e",
+   "metadata": {},
+   "source": [
+    "### Get core/accessory annotations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "6c0df670",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in expression data\n",
+    "pao1_expression_filename = paths.PAO1_COMPENDIUM\n",
+    "pa14_expression_filename = paths.PA14_COMPENDIUM\n",
+    "\n",
+    "pao1_expression = pd.read_csv(pao1_expression_filename, sep=\"\\t\", index_col=0, header=0)\n",
+    "pa14_expression = pd.read_csv(pa14_expression_filename, sep=\"\\t\", index_col=0, header=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "c51bf1f3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of PAO1 core genes: 5366\n",
+      "Number of PA14 core genes: 5363\n",
+      "Number of PAO1 core genes in my dataset: 5361\n",
```

Comment:
Why is there a difference in number of total core genes and number of core genes in your dataset? Could be worth a comment for clarity.

Reply:
Good point, I've added a comment to clarify this